index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        
+        <!-- this meta tag viewport makes sure website functions in both small and big devices -->
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Home | Karla Torres | Portfolio Website</title>
+        
+        <!-- normalize allows website to function on all browsers-->
+        <link rel="stylesheet"
+        href="http://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
+        
+        <!-- Using font-awesome website for icons-->
+        <link rel="stylesheet"
+        href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+        integrity="sha256-46qynGAkLSFpVbEBog43gvNhfrOj+BmwXdxFgVK/Kvc=" crossorigin="anonymous" />
+        <!--Using google fonts-->
+        <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@1,300&family=Viaoda+Libre&display=swap" rel="stylesheet">
+        
+        <!-- link to CSS folder and CSS file-->
+        <link rel="stylesheet" href="css/style.css">
+        
+        
+    </head>
+
+    <body>
+        <header >
+            <div class="logo">
+                <!-- <img src="img/About me Photo.JPG" alt=""> --> 
+            <button class="nav-toggle" aria-label="toggle navigation">
```

Comment:
You can't really make a `ul` a child of a `button`.  Usually you just put text inside a `button` tag.